.circleci/config.yml

```
@@ -0,0 +1,70 @@
+version: 2
+jobs:
+  build-image:
+    docker:
+      - image: docker:git
```

```suggestion
      - image: docker:20.10-git
```
Probably a good idea to pin the Docker client version?